pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Studio AsyncAPI document validation"
```

Comment:
should not titles follow some single rule, like call to action or something else

wouldn't it sound better as `Validate AsyncAPI document with Studio`?

Reply:
We thought of a sentence but it doesn't look good in the side navigation bar. We'll be getting a long sentence in two parts

Reply:
ok, but on my screen `Studio AsyncAPI document validation` is also split

<img width="315" alt="Screenshot 2022-11-22 at 14 46 25" src="https://user-images.githubusercontent.com/6995927/203329752-71fdff47-920a-4164-abc0-e544a849a280.png">

maybe we should consider making left nav wider. I kinda do not like that we need to adjust titles to fit screen, you know, it is like writing content for the browser not for the human 😄

Reply:
okay then I'll make it as Validate AsyncAPI document with Studio